app/assets/stylesheets/_item.scss

```diff
@@ -0,0 +1,31 @@
+#item-title {
+  font-weight: bold;
+  margin-top: 20px;
+  margin-left: 30px;
+  text-shadow: 0px 2px 3px #666;
+}
+
+#item-description {
+  min-height: 25px;
+  font-size: 20px;
+  margin-left: 30px;
+}
+
+#item-price {
+  min-height: 24px;
+  font-size: 20px;
+  margin-left: 30px;
+}
+
+#item-button {
+  margin-left: 30px;
+}
+
+.create-item-form {
+  width: 80%;
+}
+
+.item-title :hover {
+  color: #bfa24b;
```

Comment:
Color literals like `#bfa24b` should only be used in variable declarations; they should be referred to via variable everywhere else.